cmd/netpolicy/cmd/list.go

```diff
@@ -42,7 +45,12 @@ defined`,
 		var conns []connlist.Peer2PeerConnection
 		var err error
 
-		analyzer := connlist.NewConnlistAnalyzer(connlist.WithFocusWorkload(focusWorkload))
+		outputFormat, err := validateAndGetOutputFormat()
```

Comment:
consider using a flag validation function to the command object.
Take a look at `cmd/evaluate.go`.